packages/contracts-bedrock/src/libraries/Predeploys.sol

```diff
@@ -74,6 +74,9 @@ library Predeploys {
     /// @notice Address of the EAS predeploy.
     address internal constant EAS = 0x4200000000000000000000000000000000000021;
 
+    /// @notice Address of the SOUL_GAS_TOKEN predeploy.
```

Comment:
I guess we should use an address that avoid potential collision with OP's

Reply:
As discussed, `PREDEPLOY_COUNT` is now `4096` and the address of `SOUL_GAS_TOKEN` is the 2048th predeploy address.